ui/src/OpenSearch.tsx

```diff
@@ -228,4 +229,58 @@ export async function queryOpenSearch(question: string, index_name: string, mode
     console.log("OS query ", JSON.stringify(query))
     var response = sendQuery(query, index_name);
     return response;
+}
+
+export const createFeedbackIndex = async () => {
+    const indexMappings = {
+        "mappings": {
+            "properties": {
+                "interaction_id": {
+                    "type": "keyword"
+                },
+                "thumb": {
```

Comment:
why not make this a boolean?

Reply:
there's three states - up, down, and neutral. That simplifies the unclicking operation - instead of having to delete the document or something (handling an optional comment) I can simply set the thumb value to null

Reply:
Ah that makes sense!